accord-core/src/main/java/accord/coordinate/ExecuteEphemeralRead.java

```diff
@@ -233,6 +233,23 @@ protected boolean cancel()
             return true;
         }
 
+        @Override
+        protected void onFailure(CommitOrReadNack failReply, Throwable throwable)
```

Comment:
why duplicate this? Is this to address the double invocation of callback, so we can call super.cancel? If so, I think it would be better to fix this by changing the cancel override to instead override timeout.